.travis.yml

```
@@ -0,0 +1,27 @@
+language: generic
+os: linux
+dist: bionic
+services: docker
+
+script:
+- sudo apt-get install texlive-latex-extra texlive-lang-polish texlive-generic-recommended texlive-fonts-recommended texlive-fonts-extra pandoc make
+- make all
+
+before_deploy:
+  - git config --local user.name "Chombo Club Admin"
+  - git config --local user.email "admin@chombo.club"
+  - export TRAVIS_TAG=${TRAVIS_TAG:-$(date +'%Y-%m-%d')-$(git log --format=%h -1)}
+  - git tag $TRAVIS_TAG
+
+deploy:
+  provider: releases
+  token: $GITHUB_TOKEN
+  file_glob: true
+  file:
+    - ./build/statut.pdf
+    - ./build/reference.docx
```

```suggestion
    - ./build/statut.docx
```
I believe the output file is not called reference.docx?